diff --git a/node_modules/resolve-tspaths/dist/steps/generateChanges.js b/node_modules/resolve-tspaths/dist/steps/generateChanges.js
index e752db4..9eb6500 100644
--- a/node_modules/resolve-tspaths/dist/steps/generateChanges.js
+++ b/node_modules/resolve-tspaths/dist/steps/generateChanges.js
@@ -64,9 +64,7 @@ function replaceAliasPathsInFile(filePath, aliases, programPaths) {
     const changes = [];
     const newText = exports.MATCHERS.reduce((text, regex) => text.replace(regex, (original, importStatement, importSpecifier) => {
         // The import is an esm import if it is inside a typescript (definition) file or if it uses `import` or `export`
-        const esmImport = !filePath.endsWith(".ts") &&
-            (importStatement.includes("import") ||
-                importStatement.includes("export"));
+        const esmImport = false;
         const result = aliasToRelativePath(importSpecifier, filePath, aliases, programPaths, esmImport);
         if (!result.replacement)
             return original;
